director/director.go

```diff
@@ -894,8 +894,8 @@ func ShortcutMiddleware(defaultResponse string) gin.HandlerFunc {
 			c.Next()
 			return
 		}
-		// Regardless of the remainder of the settings, we currently handle a PUT as a query to the origin endpoint
-		if c.Request.Method == "PUT" {
+		// Regardless of the remainder of the settings, we currently handle PUT or DELETE as a query to the origin endpoint
+		if c.Request.Method == "PUT" || c.Request.Method == "DELETE" {
```

Comment:
Can you clean these up to compare against `http.MethodPut` and `http.MethodDelete`? Feel free to clean this up in other places you see them, even if it's tangentially related to what you're doing in the PR :)